scripts/scil_validate_bids.py

```diff
@@ -408,60 +404,44 @@ def main():
 
     data = []
     layout = BIDSLayout(args.in_bids, validate=False,
-                        ignore=_load_bidsignore_(args.in_bids))
+                        ignore=_load_bidsignore_(args.in_bids,
+                                                 args.bids_ignore))
     subjects = layout.get_subjects()
-
-    if args.participants_label:
-        subjects = [nSub for nSub in args.participants_label if nSub in subjects]
-
     subjects.sort()
 
-    logging.info("Found {} subject(s)".format(len(subjects)))
+    logging.warning("Found {} subject(s)".format(len(subjects)))
 
     for nSub in subjects:
         mess = '# Validating subject: {}'.format(nSub)
-        logging.info("-" * len(mess))
-        logging.info(mess)
+        logging.warning("-" * len(mess))
+        logging.warning(mess)
         dwis = associate_dwis(layout, nSub)
-
         fs_inputs = []
         t1s = []
 
         if args.fs:
-            logging.info("# Looking for FS files")
+            logging.warning("# Looking for FS files")
```

Comment:
After considerations, args.fs needs to be converted to absolute path. It does not make sense that all bids outputs are absolute and fs isn't. We can get it with : `os.path.abspath(args.fs)` (it handles the case where the path is already absolute)